File: VB/ServerSide/Controllers/WebDocumentViewerController.vb
Imports DevExpress.Web.Mvc.Controllers
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Web
Imports System.Web.Mvc

Namespace ServerSide.Controllers
	Public Class WebDocumentViewerController
		Inherits WebDocumentViewerApiController

		'
		' GET: /WebDocumentViewer/

		Public Overrides Function Invoke() As ActionResult
			Dim result = MyBase.Invoke()
			' Allow cross-domain requests.
			Response.AppendHeader("Access-Control-Allow-Origin", "*")
			Return result
		End Function

	End Class
End Namespace

File: CS/ServerSide/Controllers/WebDocumentViewerController.cs
using DevExpress.Web.Mvc.Controllers;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;

namespace ServerSide.Controllers
{
    public class WebDocumentViewerController : WebDocumentViewerApiController
    {
        //
        // GET: /WebDocumentViewer/

        public override ActionResult Invoke()
        {
            var result = base.Invoke();
            // Allow cross-domain requests.
            Response.AppendHeader("Access-Control-Allow-Origin", "*");
            return result;
        }

    }
}
